docker-build/src/test/java/io/quarkus/ts/docker/NativeConfigurationIT.java

```diff
@@ -0,0 +1,29 @@
+package io.quarkus.ts.docker;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Paths;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.EnabledOnNative;
+
+@QuarkusScenario
+@EnabledOnNative
+public class NativeConfigurationIT {
+
+    private static final String TARGET_NAME = "target";
+    private static final String NATIVE_APP_SOURCES_PARENT_FOLDER_NAME = "docker-build-1.0.0-SNAPSHOT-native-image-source-jar";
+    private static final String CUSTOM_SOURCE_FOLDER_NAME = "test";
+
+    @Test
+    public void verifyNativeSourceFolder() throws MalformedURLException, URISyntaxException {
+        URL url = Paths
+                .get(String.format("%s/%s/%s", TARGET_NAME, NATIVE_APP_SOURCES_PARENT_FOLDER_NAME, CUSTOM_SOURCE_FOLDER_NAME))
+                .toUri().toURL();
+        Assertions.assertTrue((Paths.get(url.toURI()).toFile()).exists(), "sources folder must exist on debug mode");
```

Comment:
It can be made simpler. And you don't even need to throw any checked exceptions.
```suggestion
               Path sourceFolder = Paths.get(TARGET_NAME, NATIVE_APP_SOURCES_PARENT_FOLDER_NAME, CUSTOM_SOURCE_FOLDER_NAME);
        Assertions.assertTrue(sourceFolder.toFile().exists(), "sources folder must exist on debug mode");
```